google-cloud-speech/test/google/cloud/speech/v1/speech_smoke_test.rb

```diff
@@ -0,0 +1,37 @@
+# Copyright 2017, Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# AUTO GENERATED BY GAPIC
+
+require "minitest/autorun"
+require "minitest/spec"
+require "google/cloud/speech/v1/speech_client"
+
+describe "SpeechSmokeTest" do
```

Comment:
This test makes a call to the API and fails when run an environment that does not have gcloud SDK credentials. We haven't seen this test fail on CI because of the style violations, but I believe it will fail because CI is not configured gcloud SDK credentials. 

If a smoke test it wanted, then it should be an acceptance test since it makes an API call. But it should also be written in such a way that it will work without gcloud SDK, since that will not be available on CI.